### ISSUE TITLE
[Question]: OpenCommit with Git Tower

### ISSUE THREAD

**Issue by lKingID** [di-sukharev/opencommit#419]
### Opencommit Version

3.2.2

### Node Version

22.9.0

### NPM Version

10.8.3

### What OS are you seeing the problem on?

Mac

### What happened?

In Git Tower i can't get OpenCommit to run, but in Terminal everything works

### Expected Behavior

To create a commit in git tower and open commit does work with it

### Current Behavior

OpenCommit message does not get generated in git tower

**Comment by di-sukharev** [comment]
Closing this question in favor of the actionable GUI-hook tracker #422. Git GUI applications can launch hooks with a different PATH from an interactive terminal, and this report does not include a hook configuration or error log that identifies a separate Tower-specific bug.

If Tower still fails after #422 is fixed, please open a new issue with Tower's hook log, the installed hook contents, and `which node` from both environments.
